### PR TITLE
🛠️ Fix ➾ Add back instructions on how to deploy to mainnet to the website

### PR DESCRIPTION
**Pull request by jchenche**
# 🌮 Taqueria PR

## 🪁 Description

Add back instructions on how to deploy to mainnet to the website and fixed a link issue in the website.

### CODE REVIEW

**Review by GImbrailo — APPROVED** [review]
LGTM

File: website/docs/config/config-files.mdx [hunk 1]
```diff
@@ -78,7 +78,7 @@ The settings in the `config.json` are used by various taqueria plugins. Each plu
 
 - Local Config Files:
     - Each `config.local...json` contains developer specific environment configuration. 
-    - [More details here](docs/config/environments)
```
Comment (GImbrailo) [inline website/docs/config/config-files.mdx:81]: @jchenche why this change?

**Review by GImbrailo — CHANGES_REQUESTED** [review]

File: website/docs/config/config-files.mdx (hunk 1 above)
Comment (jchenche) [reply website/docs/config/config-files.mdx:81]: to fix the link

**Review by GImbrailo — APPROVED** [review]
LGTM again 😄
